src/test/helper.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { Suspense } from 'react';
```

Comment:
this new 'test' folder will eventually be exported for consumers of the library to use